curriculum/challenges/english/blocks/zh-a1-learn-understanding-the-greetings-and-introductions/68ee3d533f5542daa3c0cf6a.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 68ee3d533f5542daa3c0cf6a
+title: Task 2
+challengeType: 19
+dashedName: task-2
+lang: zh-CN
+---
+
+<!-- (Audio) Wang Hua: 你好 (nǐ hǎo)，我是王华 (wǒ shì wáng huá)。 -->
+
+# --description--
+
+When you want to tell someone your name in Chinese, you can use a very common and natural structure:
+
+`​我是 (wǒ shì...)​​` + name.
+
+It means "I am..." or "My name is...". `我 (wǒ)` means "I". Think of `是 (shì)` as the verb "to be" in English.
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What's the name of the speaker?
+
+## --answers--
+
+`王华 (wáng huá)`
+
+---
+
+`刘明 (liú míng)`
+
+### --feedback--
+
+The audio does not mention this name.
+
+---
+
+`华王 (huá wáng)`
+
+### --feedback--
+
+The Chinese name order is family name first, so this is in a wrong order.
+
+---
+
+`你好 (nǐ hǎo)`
+
+### --feedback--
+
+This is a greeting, not the speaker's name.
+
+## --video-solution--
+
+1
+
+# --explanation--
+
+`我是 (wǒ shì...)​​` means "I am..." or "My name is...". It uses the verb `是 (shì)` after the pronoun `我 (wǒ)`, which means "I", to show identity. For example:
```

Comment:
I think the ellipsis should be moved after the Pinyin. Or should we have an ellipsis here? (GPT said Chinese uses a six-dot ellipsis, so I'll let you handle this change 😄)